docs/dev/python/table-api-users-guide/table_environment.md

```diff
@@ -0,0 +1,833 @@
+---
+title: "TableEnvironment"
+nav-parent_id: python_tableapi
+nav-pos: 27
+---
+<!--
+Licensed to the Apache Software Foundation (ASF) under one
+or more contributor license agreements.  See the NOTICE file
+distributed with this work for additional information
+regarding copyright ownership.  The ASF licenses this file
+to you under the Apache License, Version 2.0 (the
+"License"); you may not use this file except in compliance
+with the License.  You may obtain a copy of the License at
+
+  http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing,
+software distributed under the License is distributed on an
+"AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+KIND, either express or implied.  See the License for the
+specific language governing permissions and limitations
+under the License.
+-->
+
+This document is a introduction of PyFlink TableEnvironment. 
+It includes detailed descriptions of every public interface of the TableEnvironment class.
+
+* This will be replaced by the TOC
+{:toc}
+
+The Creation of TableEnvironment
```

Comment:
```suggestion
Create a TableEnvironment
```

Keep consistent with "dev/python/table-api-users-guide/intro_to_table_api.html#create-a-tableenvironment". Besides, it would be better to add a link in "http://0.0.0.0:4000/dev/python/table-api-users-guide/intro_to_table_api.html#create-a-tableenvironment" to refer to here.